core-java-modules/core-java-collections-6/src/test/java/com/baeldung/passcollectiontovarargs/PassCollectionToVarargsUnitTest.java

```diff
@@ -0,0 +1,40 @@
+package com.baeldung.passcollectiontovarargs;
+
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class PassCollectionToVarargsUnitTest {
```

Comment:
Since we are using JUnit 5 the class does not need to be public.

Reply:
public access modifier removed